python/pyarrow/orc.py

```diff
@@ -54,9 +54,25 @@ def nrows(self):
         return self.reader.nrows()
 
     @property
-    def nstripes(self):
-        """The number of stripes in the file"""
-        return self.reader.nstripes()
```

Comment:
I think you accidentally removed nstripes here

Reply:
Oops..thanks for catching this!